src/assessments/assessment-builder.tsx

```diff
@@ -239,12 +239,12 @@ export class AssessmentBuilder {
         } as Assessment;
     }
 
-    private static getStepConfig(steps: TestStep[], testStep: string): TestStep {
+    private static getStepConfig(steps: Requirement[], testStep: string): Requirement {
```

Comment:
It's _so_ tempting to suggest renaming methods like `getStepConfig` and parameters like `steps`... but this PR is plenty big already, so it makes sense to just do the single mechanical rename now to keep the immediate change as simple as possible (91 files!) and defer additional renames to future PRs.

Reply:
I agree. there's a lot more place to change, we cannot fit them in a single PR.